### DISCUSSION TITLE
Support for Bi Drectional Linking

### DISCUSSION THREAD

**Discussion by theambitiouscreative** [knowledgefutures/pubpub#1751] [Support]
Hello guys,

1. I am really interested in using this platform as a collaborative [digital garden](https://maggieappleton.com/garden-history) and I would like to know if pubpub has bi directional linking features or you're planning to offer them soon. Thank you!

**Comment by jeffpooley** [comment]
Hi @theambitiouscreative, I’m chiming in as a user. There’s no current support for automatic backlinking, a la Roam or Obsidian. There is, however, a great Connections feature, accessible in the Pub Settings “Connections” tab:

<img width="1366" alt="connections" src="https://user-images.githubusercontent.com/6372329/149509231-2b6b7fa6-4663-40ae-b035-6797426f10cd.png">

These are bi-directional, though the Connection “card” will appear at the top or bottom of a Pub depending on the “direction” specified in the Connections tab.

<img width="1366" alt="connections2" src="https://user-images.githubusercontent.com/6372329/149509438-109307c8-9679-4a82-81ec-31313f69df93.png">

**Reply by theambitiouscreative** [reply]
Thanks. The connection feature seems cool, I'll be sure to check it out!
